sdc/datatypes/hpat_pandas_stringmethods_functions.py

```diff
@@ -957,6 +957,69 @@ def _hpat_pandas_stringmethods_autogen(method_name):
     global_dict_name = 'hpat_pandas_stringmethods_{methodname}'.format(methodname=method_name)
     return _hpat_pandas_stringmethods_autogen_global_dict[global_dict_name]
 
+@overload_method(StringMethodsType, 'isspace')
+def hpat_pandas_stringmethods_isspace(self):
+    """
+        Intel Scalable Dataframe Compiler User Guide
+        ********************************************
+        Pandas API: pandas.Series.str.isspace
+
+        Limitations
+        -----------
+        Series elements are expected to be Unicode strings. Elements cannot be NaN.
+
+        Examples
+        --------
+        .. literalinclude:: ../../../examples/series/str/series_str_isspace.py
+           :language: python
+           :lines: 27-
+           :caption: Check if all the characters in the text are whitespaces
+           :name: ex_series_str_isspace
+
+        .. code-block:: console
+
+            > python ./series/str/series_str_isspace.py
+            0     True
+            1    False
+            2    False
+            3    False
+            dtype: bool
+
+        Intel Scalable Dataframe Compiler Developer Guide
+        *************************************************
+
+        Pandas Series method :meth:`pandas.core.strings.StringMethods.isspace()` implementation.
+
+        Note: Unicode type of list elements are supported only. Numpy.NaN is not supported as elements.
+
+        .. only:: developer
+
+        Test: python -m sdc.runtests sdc.tests.test_series.TestSeries.test_series_isspace_str
+
+        Parameters
+        ----------
+        self: :class:`pandas.core.strings.StringMethods`
+            input arg
+
+        Returns
+        -------
+        :obj:`pandas.Series`
+             returns :obj:`pandas.Series` object
+        """
+
+    ty_checker = TypeChecker('Method isspace().')
+    ty_checker.check(self, StringMethodsType)
+
+    def hpat_pandas_stringmethods_isspace_impl(self):
+        item_count = len(self._data)
+        result = numpy.empty(item_count, numba.types.boolean)
+        for idx, item in enumerate(self._data._data):
+            result[idx] = item.isspace()
```

Comment:
May be, it's better to cut `result = numpy.array([item.isspace() for item in self._data._data])`